index.html

```diff
@@ -31,7 +31,7 @@
                 <li><a href="">About</a></li>
                 <li><a href="">Contact</a></li>
                 <li>
-                    <a href="edubew98@gmail"><img
+                    <a href="edubew98@gmail.com"><img
```

Comment:
- [ ] As the previous reviewer mentioned, please change the  color of buttons, contrast test is still failing: 
<img width="1315" alt="Screenshot 2022-03-02 at 15 05 27" src="https://user-images.githubusercontent.com/77038610/156349999-e0e2c24c-86db-48e2-8c1d-b1b311353b8e.png">

***
I suggest using background  color for buttons provided in figma `#7D73FC` : 
<img width="1148" alt="Screenshot 2022-03-02 at 15 07 03" src="https://user-images.githubusercontent.com/77038610/156350261-7fbcd5e9-53fe-405b-9da3-480575ccd2cb.png">

You can check the contrast for your website using this link, before submitting for review please make sure that none of the contrast tests fail.

Reply:
Hi, I am using [this link](https://www.aditus.io/button-contrast-checker/), you have to insert the raw html there to test the page 🙌🏻